crates/api/src/chronicle_graphql/authorization.rs

```diff
@@ -0,0 +1,68 @@
+use base64::Engine;
+use jwtk::jwk::RemoteJwksVerifier;
+use serde_json::Value;
+use std::time::Duration;
+use thiserror::Error;
+
+#[derive(Debug, Error)]
+pub enum Error {
+    #[error("Base64 decoding failure")]
```

Comment:
Print the inner error here with {0}, it's a little infuriating you panic or have otherwise limited error reporting without the inner in the message otherwise

Reply:
Thanks, still learning how to use `thiserror`!